src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -522,7 +522,21 @@ Whether Harvesting (OAI) service is enabled
          * would also work) of never muted notifications that cannot be turned off by the users. AlwaysMuted setting overrides
          * Nevermuted setting warning is logged.
          */
-        NeverMuted
+        NeverMuted,
+        /*
+         * The following values are for the OAI server, to limit the output of 
+         * the ListIdentifiers, ListRecords and ListSets verbs to the number
+         * of records specified. Sensible defaults are going to be provided for 
+         * these by SystemConfig. 
+         */
+        oaiServerMaxListIdentifiers,
+        oaiServerMaxListRecords,
+        oaiServerMaxListSets,
+        /* 
+         * The OAI "repository name", shown by the OAI Identify verb, needs
+         * to be configurable as well. 
+         */
+        oaiServerRepositoryName
```

Comment:
**nitpick:** Please make the writing consistent and use a capital letter within the `Key` enum.

**though:** Can we pretty please avoid adding even more database settings for things that definitely are static things? #7000 got pretty far already.